### PR TITLE
Update jwt dependency

### PR DESCRIPTION
**Pull request by kibertoad**

### CODE REVIEW

**Review by lokankr — APPROVED** [review]

File: src/infrastructure/errors/errorHandler.ts [hunk 1]
```diff
@@ -7,7 +7,6 @@ import type { FreeformRecord } from '../../schemas/commonTypes'
 import { isObject, isStandardizedError } from '../typeUtils'
 
 const knownErrors = new Set([
-  'FAST_JWT_MALFORMED',
```
Comment (lokankr) [inline src/infrastructure/errors/errorHandler.ts:10]: Is this the reason for error message change as well?

Comment (kibertoad) [reply src/infrastructure/errors/errorHandler.ts:10]: yes, now we are consuming the error that fastify is throwing after catching the underlying error

Comment (kibertoad) [reply src/infrastructure/errors/errorHandler.ts:10]: wait, no, actually now we fall into this branch:
```
      const message =
        error.code === 'FST_JWT_AUTHORIZATION_TOKEN_INVALID'
          ? 'Authorization token is invalid'
          : error.message
```

Comment (kibertoad) [reply src/infrastructure/errors/errorHandler.ts:10]: so yeah, this is now handled within fastify-thrown error branch, but error message is ours

Comment (lokankr) [reply src/infrastructure/errors/errorHandler.ts:10]: Looks related https://github.com/fastify/fastify-jwt/commit/e24bea573e45d53d3b14df35e032eb8a9399124e

Comment (kibertoad) [reply src/infrastructure/errors/errorHandler.ts:10]: yup, looks like it is exactly that change, and it seems to be very intentional

**Pull request merged**